src/test/java/spark/BodyAvailabilityTest.java

```diff
@@ -38,7 +38,7 @@ public static void tearDown() {
     public static void setup() {
         LOGGER.debug("setup()");
 
-        testUtil = new SparkTestUtil(4567);
+        testUtil = new SparkTestUtil(0);
```

Comment:
It's not random. It's just mean that you doesn't care about port. I think even better to add default constructor for this case. Thank you for pointing!